x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/10_basic.yml

```diff
@@ -1232,3 +1303,63 @@ setup:
   - match: { hits.hits.3._source.k8s.pod.value.sum: 42.0 }
   - match: { hits.hits.3._source.k8s.pod.agent.id: "second" }
   - match: { hits.hits.3._source.k8s.pod.agent.version: "2.1.7" }
+
+---
+"Downsample empty and missing labels":
+  - skip:
+      version: " - 8.4.99"
```

Comment:
should this be 8.6.99 ?

Reply:
I always try to put there the minimum possible value which is `8.4.99` because in `8.5.0` we changed something around naming which also includes renaming endpoints. My understanding is that this (using the minimum possible versions) allows us to run those tests also in mixed cluster tests where there are nodes running older version of ES. Am I wrong?

Reply:
If from 8.5.0 there are no breaking changes then this skip version makes sense.

There is another aspect and that prior to 8.7.0, downsampling was tech preview. 
With that in mind, I think actually many of our tsdb yaml tests can skip versions older than that version.
But maybe we can chat about this next week.